From 1e7ed0b74692470e48670c41cfe6b8e86114911f Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Tue, 7 Jul 2026 20:53:52 +0800
Subject: [PATCH] angle-headers: reset to working commit

---
 packages/angle-headers.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/angle-headers.cmake b/packages/angle-headers.cmake
index 590aad5..32a50c1 100644
--- a/packages/angle-headers.cmake
+++ b/packages/angle-headers.cmake
@@ -3,6 +3,7 @@ ExternalProject_Add(angle-headers
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET 0cb8023c01f92b29f3738ea7472d06f8f059ed84
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone include/EGL include/KHR"
     GIT_SUBMODULES ""
-- 
2.45.2.windows.1

